rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java

```diff
@@ -233,15 +245,22 @@ public JobId submit(URL job, Map<String, String> variables, Map<String, String>
     }
 
     @Override
-    public JobId submit(File job, Map<String, String> variables)
+    public JobId submit(URL job, Map<String, String> variables, Map<String, String> genericInfos,
+            Map<String, String> headerParams)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
-        return _getScheduler().submit(job, variables);
+        return _getScheduler().submit(job, variables, genericInfos, headerParams);
     }
 
     @Override
-    public JobId submit(URL job, Map<String, String> variables)
+    public JobId submit(Map<String, String> genericInfos, URL job, Map<String, String> variables)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)